python/lambdas/pause-preservica-activity/test_pause_preservica_activity.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+from unittest.mock import patch, call
+import json
+import pause_preservica_activity
+
+secrets_manager_details = [
+    {"id": "secret-arn-1", "lambda_arn": "lambda-arn-1", "schedule_expression": "expression1"},
+    {"id": "secret-arn-2", "lambda_arn": "lambda-arn-2", "schedule_expression": "expression2"},
+]
+
+
+@pytest.fixture
+def env(monkeypatch):
+    monkeypatch.setenv('SECRETS_MANAGER_DETAILS', json.dumps(secrets_manager_details))
+    monkeypatch.setenv('ENVIRONMENT', 'test')
+
+
+@pytest.fixture
+def boto3_mocks():
+    with patch('pause_preservica_activity.eventbridge') as eventbridge_mock, \
+            patch('pause_preservica_activity.secretsmanager') as secretsmanager_mock:
+        yield eventbridge_mock, secretsmanager_mock
+
+
+def test_pause_preservica_activity(env, boto3_mocks):
```

Comment:
Is there a more descriptive name i.e. adding what we expect the this test to do?

Reply:
We expect it to pause Preservica activity. Specifically what it does is defined in the test. I don't think we gain anything by having a list of assertions in the test name which are then repeated in the body of the test.